src/components/AdminBoard.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="admin-board">
+    <reset-button
+      class="admin-button-top"
+      :activated="adminModeState[0]"
+      @reset="adminMode(0)"
+    />
+    <reset-button
+      class="admin-button-bottom"
+      :activated="adminModeState[1]"
+      @reset="adminMode(1)"
+    />
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import ResetButton from './ResetButton.vue';
+
+export default {
+  components: {
+    ResetButton,
+  },
+  data() {
+    return {
+      adminModeState: [false, false],
+    };
+  },
+  methods: {
+    adminMode(playerId) {
+      Vue.set(this.adminModeState, playerId, !this.adminModeState[playerId]);
+
+      if (this.adminModeState.every((value) => value)) {
+        console.log('Entering admin mode');
+      }
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+  @use 'styles/constants';
+  @use 'styles/colors';
+
+  #admin-board {
+    display: grid;
+    justify-items: end;
+    grid:
+    "admin-button-top" constants.$reset-button-size
+    "." auto
+    "admin-button-bottom" constants.$reset-button-size;
+
+    .admin-button-top {
+      grid-area: admin-button-top;
+    }
+
+    .admin-button-bottom {
+      grid-area: admin-button-bottom;
+    }
+
+    button.activated {
+      background-color: colors.$error;
```

Comment:
1. If we are going to set the button color in the parent component then the SideBoard should control the color of its buttons as well

2. I know this is a WIP, but I don't think the error color is the right color. I would suggest `accent-dark`. Thoughts?